cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -5198,6 +5198,124 @@ TEST(InterpreterTest, Step_Single_Memory) {
       1);
 }
 
+///////////////////////////////////////////////////////////
+// MCOPY
+
+TEST(InterpreterTest, MCOPY_PreCancun) {
+  RunInterpreterStepTest({
+      .code = {op::MCOPY},
+      .state_after = RunState::kErrorOpcode,
+      .revision = EVMC_SHANGHAI,
+  });
+}
+
+TEST(InterpreterTest, MCOPY) {
```

Comment:
name suggestion `MCOPY_Regular`  ?

Reply:
This is not the pattern in this file. 
Nevertheless, this file is going to suffer severe changes.